src/test/java/com/epam/edumanagementsystem/teacher/impl/TeacherServiceImplTest.java

```diff
@@ -84,4 +91,42 @@ void canCreateTeacher() {
     void canNotCreateTeacher() {
         assertThrows(ObjectIsNull.class, () -> teacherService.create(null));
     }
+
+    @Test
+    @DisplayName("Update Teacher positive case - given all parameters")
+    void testUpdateFields() {
+        Optional<Teacher> ofResult = Optional.of(teacher);
```

Comment:
OfResult is not a good name for a variable

Reply:
Thanks a lot, I will solve this issue